app/views/home/index.html.slim

```diff
@@ -35,7 +35,7 @@ section#sponsors
     .row
       .col-sm-12.text-center
         p RedDotRubyConf could not have become such a diverse and vibrant event, without our friendly community and our generous sponsors and supporters.
-        p If you'd like to join our dynamic group of organizations to support and engage the Ruby community, please contact us at #{link_to 'organisers@reddotrubyconf.com', 'mailto:organisers@reddotrubyconf.com'} for more information. Thank you!
+        p If you'd like to support and engage our Ruby community, please take a look at our #{link_to 'Sponsor Kit', '/rdrc2017_sponsorkit.pdf', target: "_blank"} and #{mail_to 'organisers@reddotrubyconf.com', 'get in touch with us', subject: "Sponsorship Enquiry for RedDotRubyConf 2017", target: "_blank"} for more information. Thank you!
```

Comment:
You probably want to use the `#asset_path` helper, so that it works in all environments regardless of asset pipeline configuration. 😀 

```
link_to 'Sponsor Kit', asset_path(rdrc2017_sponsorkit.pdf)
```